src/Datadog.Trace/SpanTypes.cs

```diff
@@ -36,5 +36,10 @@ public static class SpanTypes
         /// The span type for a GraphQL integration.
         /// </summary>
         public const string GraphQL = "graphql";
+
+        /// <summary>
+        /// The span type for a custom integration.
+        /// </summary>
+        public const string Custom = "custom";
```

Comment:
I didn't see where this span type was used in this PR, but it will be good to have this nonetheless.

Reply:
Oh, right. I was going to use this in WCF for non-http transports, but we decided to go with `web` for all incoming requests for UI reasons.

Reply:
It did come up recently with someone using manual instrumentation, though.

Reply:
There is a  a POV  in LATAM (TOTVS) that has run into this issue where they are not using HTTP with WCF and there are 0 Errors reported in the service summary screen even though there are errors listed.  Will this change provide visibility into TCP WCF instead of WCF only?